src/components/form/dt-location-map/dt-location-map.stories.js

```diff
@@ -162,6 +164,8 @@ function Template(args) {
       value="${JSON.stringify(value)}"
       onchange="${onchange}"
```

Comment:
Look at the other component stories and see how we used storybook actions to be able to show change events in the actions panel of Storybook